### ISSUE TITLE
5-layers loss < 1, .5R almost always 1, but test images fail

### ISSUE THREAD

**Issue by Knuust** [AlexeyAB/darknet#3967]
Hello,
i trained a 5 layer yolov3 net to my dataset and the training looks good. After ~5K Iterations the loss is already <1 without pretrained weights. Also .5R is almost always 1.
But when i try a test image with a lot of objects, the net only detects bullshit with certainty of .05.
Do you have an idea what could be the problem?
By the way: is it possible to use the standard darknet53 pretrained weights for 5-layers as well?

Thank you in advance!
Knust
